presto-main/src/test/java/com/facebook/presto/operator/scalar/TestVarbinaryFunctions.java

```diff
@@ -138,6 +140,24 @@ public void testFromHex()
     }
 
     @Test
+    public void testToBigEndian64()
+            throws Exception
+    {
+        assertFunction("to_big_endian_64(0)", VARBINARY, new SqlVarbinary(Longs.toByteArray(0)));
```

Comment:
However, I agree. For such a simple case, it'd be better to put the expected value.